config/flink.batch.conf.template

```diff
@@ -26,9 +26,10 @@ env {
 
 source {
   # This is a example input plugin **only for test and demonstrate the feature input plugin**
-  FileSource{
-    file.path = "hdfs://localhost:9000/output/text"
-    source_format = "text"
+  FileSource {
+    path = "/opt/data.txt"
```

Comment:
I think `path = "hdfs://localhost:9000/output/text"` will be better, this way can better distinguish local files from hdfs or other file system files

Reply:
Make sense, thanks